config.py

```diff
@@ -228,15 +228,23 @@ def load(cls, overrides: Optional[Dict[str, Any]] = None) -> 'Config':
         if config.project.name == "auto-detect" and project_root:
             config.project.name = project_root.name
 
-        # Resolve database path
+        # Resolve database path with proper precedence:
+        # 1. Explicit config setting (highest priority)
+        # 2. KLAUSS_DB_PATH environment variable
+        # 3. Auto-detected path based on project
         if config.database.path is None:
-            # Auto-generate: {project_name}_claude_tasks.db in klauss/
-            if klauss_dir:
-                db_filename = f"{config.project.name}_claude_tasks.db"
-                config.database.path = str(klauss_dir / db_filename)
+            # Check environment variable first
+            env_db_path = os.environ.get('KLAUSS_DB_PATH')
+            if env_db_path:
+                config.database.path = env_db_path
```

Comment:
Is the reason for using environment variables because we're using `tomli ` rather than `toml` installed so our scripts can read but not write to the config? If so, I think we'd be better off installing `toml` instead of `tomli` and leaning on our config rather than also using environment variables. That seems to add unnecessary complexity.